content/Building Engineering Goals.md

```diff
@@ -0,0 +1,66 @@
+
+### Company goals
+Some of our company goals are to keep growing and get more and bigger clients. Our engineering team needs to build its goals to align with the company’s overall vision and goals. For example, engineering delivers products with high-code quality standards but not just because our leaders are obsessed with quality. These standards are part of our vision. They link directly to our business objectives.
+
+
+## Motivation
+Last year, we decided to increase transparency and get clear about our engineering goals. We spent many hours preparing a presentation because we wanted our team to deeply understand our goals. We gave the presentation, and everything went well. We got the team’s attention, and it seemed like they understood and absorbed all the information.
+
+### Progress check
+One year is a long time in the digital industry. So we track the progress of our goals each quarter, and we rewrite some of them if necessary. At the end of the year, we officially review our progress. We’d presented our goals at the beginning of the year, and we tried to be as clear as possible. So what could go wrong?
+
+I wouldn’t say that anything went wrong. But in that year, we had a lot of growing to do. We grew up as a team not just in numbers but also in our skillsets. We launched a bunch of amazing new products. Our clients were happy, and most importantly, our team was happy too. And we learned that digital masters move fast, learn from previous experiences, and find opportunities to keep improving. What worked in the past isn’t going to work forever.
+
+
+## Improvement opportunities
+
+<img src="images/team_engagement.jpg" alt="Team engagement" width="500"/>
+
+### Increase team engagement
+Being transparent with the team about department goals is a must. A good presentation with that info works for that purpose. We encourage our team to raise their hands if they have questions, and we create opportunities to promote their participation.  
+But keeping the team involved with company goals is much more difficult. We want them to be engaged. We also want to grow their feeling of ownership for their work and their contribution to the company.
+
+**Our hypothesis.** Building goals after a couple of leader meetings and then giving a presentation to the whole team is a top-down approach. Creating a presentation without the team’s input makes them view our engineering goals as just company goals and not their own. This approach doesn’t generate a feeling of ownership.
+
+**A possible solution.** We need to engage the team in the process of developing company goals. Let their knowledge and experience contribute to deciding the best approach to any engineering problems that might arise.
```

Comment:
I'm a bit confused about this. This doesn't look like a possible solution, but more like the desired goal.

Reply:
Yes, it seems that in this section maybe we can talk about generating instances on daily basis where we encourage the team to get involved and assume the company goals as their own?